Client/Extensions/AppearanceExtensions.swift

```diff
@@ -67,14 +66,26 @@ extension AppDelegate {
         
         UIImageView.appearance(whenContainedInInstancesOf: [SettingsViewController.self])
             .tintColor = .braveLabel
-
-        UIView.appearance(whenContainedInInstancesOf: [UITableViewHeaderFooterView.self])
-            .backgroundColor = tablePrimaryColor
+        
+        //  UIView.appearance(whenContainedInInstancesOf: [UITableViewHeaderFooterView.self])
+        //    .backgroundColor = tablePrimaryColor
```

Comment:
this not needed anymore?

Reply:
Not that I saw, and it was causing issues in wallet so I removed it. I'll remove it fully instead of commenting it out before merge